libbpf-tools/tcprtt.c

```diff
@@ -225,6 +263,12 @@ int main(int argc, char **argv)
 	if (err)
 		return err;
 
+	if ((env.laddr || env.raddr)
+		&& (!memcmp(env.laddr_v6, zero_addr_v6, sizeof env.laddr_v6) || !memcmp(env.raddr_v6, zero_addr_v6, sizeof env.raddr_v6))) {
```

Comment:
Did you check when with '-a'/'-A'? if env.laddr/env.raddr is not NULL, and env.laddr_v2/raddr_v6 is NULL, the below error is triggered.
Let us use the format 'sizeof(...)' which is more uniform compared to other usage in libbpf-tools.